### ISSUE TITLE
Hax failure to load libraries due to cache rootpy overload

### ISSUE THREAD

**Issue by jhowl01** [XENON1T/pax#532]
It is likely that many already know how to avoid this, but when trying to make minitrees, errors like this:
```
cling::DynamicLibraryManager::loadLibrary(): libopenblasp-r0-39a31c03.2.18.so: cannot open shared object file: No such file or directory
…
dask.async.RuntimeError: failed to load the library for 'std::vector<Hit>' @ 119a4adf77b59cb0
```
will occur if you do not regularly clear the `.cache/rootpy` folder in your home directory and remove `pax_event_class_*` files.

**Comment by JelleAalbers** [comment]
Yes, this is a known recurrent issue with our Rube Goldberg python/root/python/root/rootpy/pyroot system, but good to have it open.

**Comment by JelleAalbers** [comment]
In case anyone ends up here with this problem, we've settled upon a cleansing ritual described here: https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:beginnersguide#debugging_root_compilation_problems_with_haxminitreesload

We still have no idea what causes the issue or how to reproduce it. Perhaps it has the same root cause (pun intended) as #629
